src/recon_buildblock/PoissonLogLikelihoodWithLinearModelForMeanAndListModeData.cxx

```diff
@@ -372,14 +374,6 @@ set_up_before_sensitivity(shared_ptr <const TargetT > const& target_sptr)
         return Succeeded::no;
     }
 
-  if (this->normalisation_sptr->set_up(
-                this->list_mode_data_sptr->get_exam_info_sptr(), proj_data_info_sptr->create_shared_clone()) == Succeeded::no)
-    {
-      warning("PoissonLogLikelihoodWithLinearModelForMeanAndListModeDataWithProjMatrixByBin: "
-              "set-up of normalisation failed.");
-      return Succeeded::no;
-    }
-
```

Comment:
need to review why this was removed

Reply:
It is now in `PoissonLogLikelihoodWithLinearModelForMeanAndListModeDataWithProjMatrixByBin` as it needs the "sensitivity" projector.